### DISCUSSION TITLE
Simple tractography from eigenvectors

### DISCUSSION THREAD

**Discussion by brechmos** [dipy/dipy#2652] [Tractography / Tractometry]
I have eigenvalues and eigenvectors calculated outside of `dipy` and would like to use `dipy` to do the tractography. 

Is there example code that shows how to load up eigenvector data and do tractography without needing a gradient table, b-values, b-vectors etc?

Thanks.

**Comment by skoudoro** [comment]
Hi @brechmos,

Unfortunately,  no example to show how to do it this way.  It is clearly possible with many functions already made in DIPY, but can be a bit tricky.  you need to look at `peaks_from_model` method, create a new custom and simple model that loads you eigenvector/values and you should be ready to go.

But clearly, this is something that we need to create such a tutorial.

**Reply by brechmos** [reply]
@skoudoro Thanks for the hint. If there is a chance someone can supply an outline of this (with some detail :) ), I have some data that I can extend and test. I think some help on the custom model (that loads the eigenvector and eigenvalues) would be helpful.

**Reply by skoudoro** [reply]
Not sure I will have time, but you can book at slot during [DIPY OFFICE hours](https://docs.google.com/spreadsheets/d/1IVaixEbjKQqS2MrZHw1L2Gso8a0GannT19GwWYhwFc0/edit#gid=0) 

Me or someone else will be able to spend time with you on this feature during this slot.
